README.md

```diff
@@ -143,58 +143,55 @@ Read the full documentation on [ReadTheDocs](https://pymove.readthedocs.io/en/la
 
 PyMove **proposes**:
 
--   A familiar and similar syntax to Pandas;
--   Clear documentation;
--   Extensibility, since you can implement your main data structure by manipulating other data structures such as Dask DataFrame, numpy arrays, etc., in addition to adding new modules;
--   Flexibility, as the user can switch between different data structures;
--   Operations for data preprocessing, pattern mining and data visualization.
+- A familiar and similar syntax to Pandas;
+- Clear documentation;
+- Extensibility, since you can implement your main data structure by manipulating other data structures such as Dask DataFrame, numpy arrays, etc., in addition to adding new modules;
+- Flexibility, as the user can switch between different data structures;
+- Operations for data preprocessing, pattern mining and data visualization.
 
 ---
 
 ## Creating a Virtual Environment
 
 It is recommended to create a virtual environment to use pymove. Requirements: Anaconda Python distribution installed and accessible
 
-1.  In the terminal client enter the following where `yourenvname` is the name you want to call your environment, and replace `x.x` with the Python version you wish to use. (To see a list of available python versions first, type conda search "^python$" and press enter.)
-    -   `conda create -n <yourenvname> python=x.x`
-    -   Press y to proceed. This will install the Python version and all the associated anaconda packaged libraries at `path_to_your_anaconda_location/anaconda/envs/yourenvname`
+1. In the terminal client enter the following where `env_name` is the name you want to call your environment, and replace `x.x` with the Python version you wish to use. (To see a list of available python versions first, type conda search "^python$" and press enter.)
+    - `conda create -n <env_name> python=x.x`
+    - Press y to proceed. This will install the Python version and all the associated anaconda packaged libraries.
 
-2.  Activate your virtual environment. To activate or switch into your virtual environment, simply type the following where yourenvname is the name you gave to your environment at creation.
-    -   `conda activate <yourenvname>`
+2. Activate your virtual environment. To activate or switch into your virtual environment, simply type the following.
+    - `conda activate <env_name>`
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6295868)